frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -13,9 +13,11 @@ import {
   Title,
 } from '@patternfly/react-core';
 import {
+  COMMUNITY_PROVIDERS_WARNING_CONFIGMAP_KEY,
```

Comment:
nit: you could use 
COMMUNITY_PROVIDERS_WARNING_CONFIGMAP_KEY as mapKey,
... as localStoreKey,